PhotoGether/DomainLayer/PhotoGetherDomain/PhotoGetherDomain/UseCaseImpl/CaptureVideosUseCaseImpl.swift

```diff
@@ -4,21 +4,51 @@ import DesignSystem
 
 public final class CaptureVideosUseCaseImpl: CaptureVideosUseCase {
     public func execute() -> [UIImage] {
-        guard let localImage = connectionRepository.capturedLocalVideo
-        else { return [
-            PTGImage.temp1.image,
-            PTGImage.temp2.image,
-            PTGImage.temp3.image,
-            PTGImage.temp4.image]
-        }
-        let remoteImages = connectionRepository.clients.map { $0.captureVideo() }
+        let localImage = connectionRepository.capturedLocalVideo ?? UIImage()
+        
+        let localUserImageInfo = [(connectionRepository.localUserInfo?.viewPosition, localImage)]
+        let remoteUserImagesInfo = connectionRepository.clients
+            .map { ($0.remoteUserInfo?.viewPosition, $0.captureVideo()) }
         
-        return [localImage] + remoteImages
+        return sortImages(localUserImageInfo + remoteUserImagesInfo)
     }
     
     private let connectionRepository: ConnectionRepository
     
     public init(connectionRepository: ConnectionRepository) {
         self.connectionRepository = connectionRepository
     }
+    
+    private func sortImages(_ images: [(viewPosition: UserInfo.ViewPosition?, image: UIImage)]) -> [UIImage] {
+        let convertedArray = images.map {
+            (position: PositionOder(rawValue: $0.viewPosition?.rawValue ?? -1),
+             image: $0.image)
+        }
+        
+        // 배열의 2번 인덱스가 마지막 자리이기 때문에 nil일 경우 2로 설정했습니다
+        let sortedByViewPosition = convertedArray.sorted {
+            let lhs = $0.position?.sequence ?? 2
+            let rhs = $1.position?.sequence ?? 2
+            return lhs < rhs
+        }
+        
+        return sortedByViewPosition.map { $0.image }
+    }
+}
+
+// case의 순서는 참가자의 참가 순서에 따른 화면 배치이고 sequence는 이미지 데이터 전달할 때의 배열 순서입니다
```

Comment:
```suggestion
/// case의 순서는 참가자의 참가 순서에 따른 화면 배치이고 sequence는 이미지 데이터 전달할 때의 배열 순서입니다
```

description으로 달아두면 나중에 해당 enum이 어떤 용도인지 opt로 보는데 좋을것 같습니다 👍

Reply:
[efea361](https://github.com/boostcampwm-2024/iOS04-PhotoGether/pull/179/commits/efea3619587cf44453cac9805561dd1bbe4193a5) Suggestion 감사합니다!